wagtail/users/views/users.py

```diff
@@ -219,16 +186,7 @@ def get_list_buttons(self, instance):
         list_buttons = []
 
         for hook in hooks.get_hooks("register_user_listing_buttons"):
-            if accepts_kwarg(hook, "request_user"):
-                hook_buttons = hook(user=instance, request_user=self.request.user)
-            else:
-                # old-style hook that accepts a context argument instead of request_user
-                hook_buttons = hook(RequestContext(self.request), instance)
-                warn(
-                    "`register_user_listing_buttons` hook functions should accept a `request_user` argument instead of `context` -"
-                    f" {hook.__module__}.{hook.__name__} needs to be updated",
-                    category=RemovedInWagtail70Warning,
-                )
+            hook_buttons = hook(user=instance, request_user=self.request.user)
```

Comment:
Oh, I forgot to say. We might want to get https://github.com/wagtail/wagtail/issues/11915 fixed (PR in https://github.com/wagtail/wagtail/pull/12301, will rebase) as part of the release